webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala

```diff
@@ -485,14 +391,12 @@ final case class ProjectsResponderADMLive(
    */
   override def projectKeywordsGetRequestADM(projectIri: Iri.ProjectIri): Task[ProjectKeywordsGetResponseADM] =
     for {
-      id <- IriIdentifier
-              .fromString(projectIri.value)
-              .toZIO
-              .mapError(e => BadRequestException(e.getMessage))
-      keywords <- getSingleProjectADM(id)
+      id <- IriIdentifier.fromString(projectIri.value).toZIO.mapError(e => BadRequestException(e.getMessage))
+      keywords <- projectService
+                    .findProjectKeywordsBy(id)
                     .flatMap(ZIO.fromOption(_))
-                    .mapBoth(_ => NotFoundException(s"Project '${projectIri.value}' not found."), _.keywords)
-    } yield ProjectKeywordsGetResponseADM(keywords)
+                    .orElseFail(NotFoundException(s"Project '${projectIri.value}' not found."))
+    } yield keywords
```

Comment:
I don't understand why we don't need `ProjectKeywordsGetResponseADM(keywords)` anymore and `keywords` is enough.

Reply:
Because we are now able to directly invoke the respective method on the `ProjectADMService` which already does the work for us and returns the `ProjectKeywordsGetResponseADM`.